src/url_helper.rs

```diff
@@ -30,9 +37,9 @@ mod tests {
 
     #[test]
     fn url_to_path() {
-        let str = super::to_path(&Url::parse("https://lwn.net/Kernel/").unwrap());
+        let str = super::to_path(&Url::parse("https://lwn.net/Kernel/#fragment").unwrap());
```

Comment:
It would be nice to have two tests for this. Once with the fragment, and one without.